src/Kiota.Builder/Refiners/PhpRefiner.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Kiota.Builder.Extensions;
+
+namespace Kiota.Builder.Refiners
+{
+    public class PhpRefiner: CommonLanguageRefiner
+    {
+        public PhpRefiner(GenerationConfiguration configuration) : base(configuration) { }
+        
+        public override void Refine(CodeNamespace generatedCode)
+        {
+            //AddInnerClasses(generatedCode);
+            AddDefaultImports(generatedCode, defaultUsingEvaluators);
+            MakeModelPropertiesNullable(generatedCode);
```

Comment:
does php support union types?

Reply:
Not in the code but in the doc comments. @baywet

Reply:
so how would a union type be handled at compile time, and at runtime? do you pass anything you want because the type is a bit more dynamic than most languages and then test for what you received?

Reply:
We don't have union types until PHP 8. So in the case of PHP below 8, we rely dynamic type resolution.

Reply:
hence my comment, I believe the generation should rely on wrapper types like Go/Java/CSharp do today.